src/index.css

```diff
@@ -1,6 +1,12 @@
 html {
   font-size: 62.5%; /* 62.5% of 16px = 10px */
 }
+
+a {
```

Comment:
No. Look, that's the purpose of this css code - instead of reseting link default styles(blue color with underline) individually in every file, just do it once on a global level